packages/react/src/components/Popup/createPopperReferenceProxy.ts

```diff
@@ -0,0 +1,37 @@
+import { isRefObject, toRefObject } from '@stardust-ui/react-component-ref'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+class ReferenceProxy {
```

Comment:
is there any interface provided by PopperJS, so that we would have clear errors in case if ours violates it?

Reply:
Yep, add `implements PopperJS.ReferenceObject`